config/deploy.rb

```diff
@@ -36,3 +36,14 @@
 
 # Default value for keep_releases is 5
 # set :keep_releases, 5
+
+after 'deploy:migrate', 'seed_catalog'
+
+desc 'seed the catalog'
+task :seed_catalog do
+  on roles(:all) do
+    within current_path do
+      execute :rake, 'seed_catalog'
```

Comment:
i think this should `rake db:seed`, cause `seed_catalog` will seed the storage roots. `rake db:seed` will seed the Endpoints, Preservation Policy, and EndpointType

Reply:
D'oh